crates/biome_js_type_info/src/local_inference.rs

```diff
@@ -737,13 +741,16 @@ impl TypeData {
             },
             AnyTsType::TsTypeofType(ty) => Self::from_ts_typeof_type(resolver, scope_id, ty),
             AnyTsType::TsUndefinedType(_) => Self::Undefined,
-            AnyTsType::TsUnionType(ty) => Self::union_of(
-                ty.types()
+            AnyTsType::TsUnionType(ty) => {
+                let types = ty
+                    .types()
                     .into_iter()
                     .filter_map(|ty| ty.ok())
```

Comment:
Wouldn't `flatten` do the same without using `ok()`?

Reply:
Yep, good shout!